modules/microservice/main.tf

```diff
@@ -480,6 +481,9 @@ resource "azurerm_app_service" "microservice" {
     min_tls_version = "1.2"
     #dotnet_framework_version = "v5.0"
     #websockets_enabled = true # Will need for Blazor hosted appservice
+    cors {
+      allowed_origins = var.allowed_origins
```

Comment:
Should we make these blocks dynamic?

Also it looks like this one should be from local instead of var?

Reply:
Good call out on the var vs local.

Reply:
As for making these blocks dynamic, I had that during my first go-around.  Found that if the cors block is not specified (when the dynamic is not run / for_each is an empty list) then it leaves the previous values in there and doesn't remove / update.  With this, the values will always match what is defined.